app/controllers/finders_controller.rb

```diff
@@ -98,6 +101,35 @@ def raw_finder
     @raw_finder ||= finder_api.content_item_with_search_results
   end
 
+  def results_for_feed
+    @results_for_feed ||= result_set_presenter_class.new(
+      finder_for_feed,
+      filter_params,
+      sort_presenter,
+      content_item.metadata_class,
+      show_top_result?,
+      debug_score?,
+    )
+  end
+
+  def finder_for_feed
+    @finder_for_feed ||= begin
+      finder_api_for_feed = finder_api_class.new(
+        content_item.as_hash,
+        filter_params,
+        override_sort_for_feed: true
+      )
+
+      raw_finder_for_feed = finder_api_for_feed.content_item_with_search_results
+
+      finder_presenter_class.new(
+        raw_finder_for_feed,
+        finder_api_for_feed.search_results,
+        filter_params,
+      )
```

Comment:
Could you instead do something like this, rather than having separate methods for the feed in the controller?

```
  def finder_api(is_atom_feed: false)
    @finder_api ||= finder_api_class.new(content_item.as_hash, filter_params, override_sort_for_feed: is_atom_feed)
  end
```

Reply:
I think that the AtomPresenter shouldn't need a FinderPresenter, and instead should be able to render search results from the FinderApi alone. But that's a bit out of scope for this PR!